modules/serverless-rn-1-29-1.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-1_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29.1
+
+{ServerlessProductName} 1.29.1 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-1_{context}"]
+== New features
```

Comment:
I'm not sure we should add "New features" section for these release notes. Serverless 1.29.1 doesn't include any new features it only has bug fixes. Mentioning all the versions below might suggest that these are new. But they're same as in 1.29.0.